src/core/crypto.js

```diff
@@ -0,0 +1,27 @@
+
+import crypto from 'crypto';
+
+const algorithm = 'aes-256-cbc';
+
+export const DEFAULT_KEY = '776fe87eec8c9ba8417beda00b23cf22f5e134d9644d0a195cd9e0b7373760c1';
```

Comment:
is there a plan to allow this key to be injected rather than hard-coding it?

Reply:
I can  setup some env variables on the sdk, let me check